src/codex_plus/hooks.py

```
@@ -0,0 +1,964 @@
+                        exec_path = p
+                        break
+            else:
+                exec_path = exec0
+
+            # Determine interpreter if needed
+            if exec_path:
+                # Replace first token with resolved path
+                parts[0] = exec_path
+                is_exe = _os.access(exec_path, _os.X_OK)
+                if exec_path.endswith(".py") and not is_exe:
+                    parts = ["python3", exec_path] + parts[1:]
```

Ruff (0.13.1)

Consider `["python3", exec_path, *parts[1:]]` instead of concatenation

Replace with `["python3", exec_path, *parts[1:]]`

(RUF005)